docs/aws/README.md

```diff
@@ -3,31 +3,32 @@
 This guide will provide a complete Karpenter installation for AWS.
 These steps are opinionated and may need to be adapted for your use case.
 
+> This guide should take less than 1 hour to complete and should cost less than $.25
+
 ## Environment
 ```bash
-CLOUD_PROVIDER=aws
+export CLUSTER_NAME=$USER-karpenter-demo
+export AWS_DEFAULT_REGION=us-west-2
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-CLUSTER_NAME=$USER-karpenter-demo
-AWS_DEFAULT_REGION=us-west-2
+LATEST_KARPENTER_VERSION=$(curl -fsSL \
+  https://api.github.com/repos/awslabs/karpenter/releases/latest \
+  | jq -r '.tag_name')
 ```
 
 ### Create a Cluster
 
-Create an EKS cluster
+Karpenter can run anywhere, including on self-managed node groups, [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), or [AWS Fargate](https://aws.amazon.com/fargate/).
+This demo will run Karpenter on Fargate, which means all EC2 instances added to this cluster will be controlled by Karpenter autoscaling.
+
 ```bash
-eksctl create cluster \
---name ${CLUSTER_NAME} \
---node-type m5.large \
---nodes 1 \
---nodes-min 1 \
---nodes-max 10 \
---managed \
---with-oidc
+curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/"${LATEST_KARPENTER_VERSION}"/docs/aws/eks-config.yaml \
```

Comment:
Doesn't this work 
```
curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/$LATEST_KARPENTER_VERSION/docs/aws/eks-config.yaml
```

Reply:
Until this is merged you'll have to change the curl command to `cat ./docs/aws/eks-config.yaml` on this PR.

Reply:
Or you should be able to
```
curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/7afb5fdc65373e96efe08edcb52837673cd5c9b6/docs/aws/eks-config.yaml
```